pkg/porter/show.go

```diff
@@ -31,9 +31,9 @@ func (so *ShowOptions) Validate(args []string, cxt *context.Context) error {
 	return so.ParseFormat()
 }
 
-// ShowInstances shows a bundle, or more properly a bundle claim, along with any
+// ShowInstallations shows a bundle, or more properly a bundle claim, along with any
```

Comment:
Should we update to `or more properly a bundle installation`?  Not sure how much we're wanting to update legacy instances of the term `claim`, which certainly has meaning in the CNAB spec(s) but perhaps is mostly intended to be replaced by `installation` in Porter.  Granted, this is a function comment/description, so quite minor.

Reply:
I think we should limit the scope of this PR to just updating the function names, fields and user facing text. Let's leave the explanatory comments to the maintainers to update because I think the changes will be nuanced and having someone else update them will be a game of telephone.

So don't worry too much about finding the word claim, I'll add a follow-up issue to do another pass as part of my claim milestone for the v0.28 release.

Reply:
https://github.com/deislabs/porter/issues/1105